domino/routes.py

```diff
@@ -291,5 +289,31 @@ def find_project_by_owner_name_and_project_name_url(self):
         )
 
     # User URLs
-    def users_get(self):
+    def users_get(self) -> str:
         return self.host + "/v4/users"
+
+    # Budgets and Billing Tags
+    def budget_settings(self) -> str:
+        return self.host + "/v4/cost/budgets/global/alertsSettings"
```

Comment:
Is it possible to use the public endpoints instead of the v4 endpoints?

Reply:
most of the previously written code is using v1/v4, Perhaps we should refactor all of them.